control_msgs/msg/MultiDOFJointTolerance.msg

```diff
@@ -0,0 +1,28 @@
+# The tolerances for a multi-dof joint
+
+# Since each joint has multiple degrees of freedom,
+# there can be multiple tolerances for each joint, each looking
+# at different components.
+
+# If the component is X_AXIS, Y_AXIS, or Z_AXIS, then the tolerance
+# is only applied for the specific axis.
+# However, if the component is TRANSLATION, then the tolerance is for
+# the overall Euclidean distance.
+# For these components, the units are meters, meters/sec and meters/sec^2.
+# Z_AXIS is only valid with a floating joint, not planar.
+
+# If the component is ANGULAR the tolerance is measured in
```

Comment:
ANGULAR doesn't exist. I think you mean ROTATION at L22?

Reply:
Good catch